innersvg.js

```diff
@@ -48,50 +48,54 @@ var serializeXML = function(node, output) {
   }
 }
 // The innerHTML DOM property for SVGElement.
-Object.defineProperty(SVGElement.prototype, 'innerHTML', {
-  get: function() {
-    var output = [];
-    var childNode = this.firstChild;
-    while (childNode) {
-      serializeXML(childNode, output);
-      childNode = childNode.nextSibling;
-    }
-    return output.join('');
-  },
-  set: function(markupText) {
-    // Wipe out the current contents of the element.
-    while (this.firstChild) {
-      this.removeChild(this.firstChild);
-    }
-
-    try {
-      // Parse the markup into valid nodes.
-      var dXML = new DOMParser();
-      dXML.async = false;
-      // Wrap the markup into a SVG node to ensure parsing works.
-      var sXML = '<svg xmlns=\'http://www.w3.org/2000/svg\' xmlns:xlink=\'http://www.w3.org/1999/xlink\'>' + markupText + '</svg>';
-      var svgDocElement = dXML.parseFromString(sXML, 'text/xml').documentElement;
-
-      // Now take each node, import it and append to this element.
-      var childNode = svgDocElement.firstChild;
-      while(childNode) {
-        this.appendChild(this.ownerDocument.importNode(childNode, true));
+if (!SVGElement.prototype.innerHTML) {
```

Comment:
This throws ```VM16:1 Uncaught TypeError: Illegal invocation```

Reply:
@MariMax I can't really tell what issue you were trying to solve with this check. Can you confirm if ```Object.hasOwnProperty(SVGElement.prototype, 'innerHTML')``` will achieve the same thing? It doesn't throw the illegal invocation error.

Reply:
We should check
```
'innerHTML' in SVGElement.prototype
```
to see if innerHTML is defined for SVG elements and not to override this default property

Reply:
I like that better than hasOwnProperty. I'll open a PR.

Reply:
sorry, It was silly correct way to do it would be 
`'innerHTML' in SVGElement.prototype` will make a new pr

Reply:
@MariMax @dnozay https://github.com/dnozay/innersvg-polyfill/pull/7 fixes the illegal invocation error.

Reply:
@dnozay also happy to help with maintenance of this if you want to add me to npm - username is willyboy.